site/content/languages/tinygo.md

```diff
@@ -145,77 +145,38 @@ calls a free function, with the same `ptr`, afterwards and unconditionally.
 
 Note: wazero includes an [example project][8] that shows this.
 
-There are two ways to implement this pattern, and they affect how to implement
-the `ptrToString` function above:
-* Built-in `malloc` and `free` functions
-* Custom `malloc` and `free` functions
-
-While both patterns are used in practice, TinyGo maintainers only support the
-custom approach. See the following issues for clarifications:
-* [WebAssembly exports for allocation][9]
-* [Memory ownership of TinyGo allocated pointers][10]
-
-#### Built-in `malloc` and `free` functions
-
-The least code way to allow the host to allocate memory is to call the built-in
-`malloc` and `free` functions exported by TinyGo:
+The general call patterns are the following. Host is the process embedding the
+WebAssembly runtime, such as wazero. Guest is the Tinygo source compiled to
```

Comment:
Assuming the README title is the expected spelling https://github.com/tinygo-org/tinygo#tinygo---go-compiler-for-small-places
```suggestion
WebAssembly runtime, such as wazero. Guest is the TinyGo source compiled to
```